### PR TITLE
Feat: use send instead of submit and watch

### LINKED ISSUE #1585: Try out ".send" instead of ". submitAndWatchExtrinsic"
`send` allows to pass a callback, and will give back all the status, events right away, without the need to subscribe

@Tbaut suggested to try `send` - [link](https://github.com/novasamatech/nova-spektr/pull/1547)
Documentation [here](https://polkadot.js.org/docs/api/start/api.tx.subs#transaction-events) & [here](https://polkadot.js.org/docs/api/cookbook/tx#how-do-i-get-the-decoded-enum-for-an-extrinsicfailed-event)

```javascript
.send(sender, ({ status, events }) => {
    if (status.isInBlock || status.isFinalized) {
      events
        // find/filter for failed events
        .filter(({ event }) =>
          api.events.system.ExtrinsicFailed.is(event)
        )
        // we know that data for system.ExtrinsicFailed is
        // (DispatchError, DispatchInfo)
        .forEach(({ event: { data: [error, info] } }) => {
          if (error.isModule) {
            // for module errors, we have the section indexed, lookup
            const decoded = api.registry.findMetaError(error.asModule);
            const { docs, method, section } = decoded;

            console.log(`${section}.${method}: ${docs.join(' ')}`);
          } else {
            // Other, CannotLookup, BadOrigin, no extra info
            console.log(error.toString());
          }
        });
    }
```

### PR DESCRIPTION
**Pull request by Asmadek**
* use `signAndSend` for sign operation

### CODE REVIEW

**Review by pgolovkin — APPROVED** [review]

File: src/renderer/entities/transaction/lib/transactionService.ts [hunk 1]
```diff
@@ -35,6 +35,18 @@ import {
 } from './common/types';
 import { walletUtils } from '../../wallet';
 
+class RawSigner implements Signer {
+  signature: HexString;
+
```
Comment (sokolova-an) [inline src/renderer/entities/transaction/lib/transactionService.ts]: Why do we have to use classes with signer? Is there no way to do it without it?

**Review by sokolova-an — APPROVED** [review]

File: src/renderer/entities/transaction/lib/transactionService.ts [hunk 2]
```diff
@@ -59,73 +71,40 @@ async function getTransactionFee(transaction: Transaction, api: ApiPromise): Pro
...
 
-      const signedBlock = await api.rpc.chain.getBlock();
-      const blockHeight = signedBlock.block.header.number.toNumber();
-      const apiAt = await api.at(signedBlock.block.header.hash);
-      const allRecords = await apiAt.query.system.events();
+  const options = { signer: new RawSigner(signature), ...unsigned };
 
-      let actualTxHash = result.inner;
+  extrinsic
+    // @ts-ignore
```
Comment (Tbaut) [inline src/renderer/entities/transaction/lib/transactionService.ts]: Can we add a comment to explain why it's ok to ignore TS errors here?

File: src/renderer/entities/transaction/lib/common/types.ts [hunk 3]
```diff
@@ -22,13 +22,6 @@ import { DecodedTransaction, Transaction, TransactionType } from '@entities/tran
 // =====================================================
 
 export type ITransactionService = {
-  createPayload: (
```
Comment (tuul-wq) [inline src/renderer/entities/transaction/lib/common/types.ts:25]: 👍

File: src/renderer/entities/transaction/lib/transactionService.ts (hunk 2 above)
Comment (tuul-wq) [reply src/renderer/entities/transaction/lib/transactionService.ts]: The main issue is
```javascript
TS2322: Type `0x${string}` is not assignable to type number | IExtrinsicEra | undefined
```
I also wonder how to correctly convert era, or should it be skipped at all 🤔

**Review by tuul-wq — APPROVED** [review]

File: src/renderer/entities/transaction/lib/transactionService.ts (hunk 1 above)
Comment (tuul-wq) [reply src/renderer/entities/transaction/lib/transactionService.ts]: Moved to a separate file
```javascript
const options: Partial<SignerOptions> = {
    era: api.registry.createType('ExtrinsicEra', unsigned.era),
    signer: new RawSigner(signature),
    assetId: unsigned.assetId,
    tip: unsigned.tip,
    nonce: unsigned.nonce,
    blockHash: unsigned.blockHash,
  };
```
`signer` expects something that implements `Signer` interface so the easiest way is to create a class
@Asmadek anyway, can discuss another solution

**Review by sokolova-an — APPROVED** [review]

**Pull request merged**